contrib/pyln-proto/pyln/proto/message/__init__.py

```diff
@@ -10,6 +11,9 @@
     "SubtypeType",
     "FieldType",
     "split_field",
+    "SizedArrayType",
+    "DynamicArrayType",
+    "EllipsisArrayType",
```

Comment:
Are these only required for unit tests? In that case we could also not expose them here, and have the unit tests reach directly into `.array_types`

```python
from pyln.proto.message.array_types import SizedArrayType, DynamicArrayType, EllipsisArrayType
```

If that helps keep the interface clean for the actual protocol tests.